### PR TITLE
DOC-602: add example for disable table toolbar and fix examples

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-602

Description of Changes:
* Clarify disable `table_toolbar` with example

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: plugins/table.md [hunk 1]
```diff
@@ -27,23 +28,38 @@ These settings affect the execution of the `table` plugin and let you modify the
 
 ### `table_toolbar`
 
-This option allows you to specify the buttons and the order that they will appear on within {{site.productname}}'s inline contextual toolbar for tables. To specify the controls that should appear on {{site.productname}}'s table toolbar, the `table_toolbar` option should be provided with a space separated list of toolbar controls. To create groups within this list, please add `|` pipe characters between the groups of controls that you would like to create. If you want to disable this toolbar simply set the value to an empty string.
+This option allows you to specify the buttons and the order that they will appear on within {{site.productname}}'s inline contextual toolbar for tables. To specify the controls that should appear on {{site.productname}}'s table toolbar, the `table_toolbar` option should be provided with a space separated list of toolbar controls. To create groups within this list, please add `|` pipe characters between the groups of controls that you would like to create.
```
Comment (metricjs) [inline plugins/table.md:31]: "appear on within"?

File: plugins/table.md [hunk 2]
```diff
@@ -27,23 +28,38 @@ These settings affect the execution of the `table` plugin and let you modify the
...
+
+To disable the table toolbar, set the value to an empty string.
 
 **Type:** `String`
 
-**Default Value:** `"tableprops tabledelete | tableinsertrowbefore tableinsertrowafter tabledeleterow | tableinsertcolbefore tableinsertcolafter tabledeletecol"`
+**Default Value:** `'tableprops tabledelete | tableinsertrowbefore tableinsertrowafter tabledeleterow | tableinsertcolbefore tableinsertcolafter tabledeletecol'`
 
-**Possible Values:**  `table`, `tablerowprops`, `tablecellprops`
+**Possible Values:**  `table`, `tablerowprops`, `tablecellprops`, `tableprops`, `tabledelete`, `tableinsertrowbefore`, `tableinsertrowafter` `tabledeleterow`, `tableinsertcolbefore`, `tableinsertcolafter`, `tabledeletecol`
```
Comment (lnewson) [inline plugins/table.md:39]: There's more available here. It technically allows all toolbar button types, so maybe we should link to that instead?

File: plugins/table.md (hunk 1 above)
Comment (tylerkelly13) [reply plugins/table.md]: Fixed? by https://github.com/tinymce/tinymce-docs/pull/1580/commits/43b982fe79a375e25517c44e0c01f3409f445478

File: plugins/table.md (hunk 2 above)
Comment (tylerkelly13) [reply plugins/table.md]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1580/commits/43b982fe79a375e25517c44e0c01f3409f445478

**Pull request merged**